evalai/utils/teams.py

```diff
@@ -51,22 +63,31 @@ def display_participant_teams():
 
     teams = response["results"]
     if len(teams) != 0:
-        pretty_print_team_data(teams)
+        pretty_print_team_data(teams, is_host)
     else:
         echo("Sorry, no teams found!")
 
 
-def create_participant_team(team_name):
+def create_team(team_name, team_url, is_host):
     """
     Function to create a new team by taking in the team name as input.
     """
-    url = "{}{}".format(get_host_url(), URLS.participant_team_list.value)
+    url = "{}{}"
+
+    if is_host:
+        url = url.format(get_host_url(), URLS.create_host_team.value)
+    else:
+        url = url.format(get_host_url(), URLS.participant_team_list.value)
 
     headers = get_request_header()
     headers['Content-Type'] = 'application/json'
 
     data = {}
-    data["team_name"] = team_name
+    if team_url:
+        data["team_name"] = team_name
+        data["team_url"] = team_url
+    else:
+        data["team_name"] = team_name
```

Comment:
I don't quite see the point of this else statement, since `data["team_name"] = team_name` is independent of the `if` condition.

Reply:
Careless mistakes. :sweat_smile: